WISE-Community/WISE-Client
# Manage Students: Change Password

## Changes
Added option for teacher to change student password from the Manage Students section.

Closes #291. 

## Test
- Pull the WISE-API branch issue-25-teacher-change-student-password from WISE-Community/WISE-API#25 to begin testing.
- Make sure changing student password works.
- Make sure Google icon appears instead of change password button for students that sign in with Google.
